src/Symfony/Bundle/FrameworkBundle/Tests/Console/Descriptor/AbstractDescriptorTest.php

```diff
@@ -69,6 +69,17 @@ public function getDescribeContainerBuilderTestData()
         return $this->getContainerBuilderDescriptionTestData(ObjectsProvider::getContainerBuilders());
     }
 
+    /** @dataProvider getDescribeContainerExistingClassDefinitionTestData */
```

Comment:
```
/**
 * @dataProvider ...
 */
```

Reply:
Are you sure ? Its inlined on all tests of this class.

Reply:
Oh.. it seems this is the only file that does it: https://github.com/symfony/symfony/search?q=dataProvider&unscoped_q=dataProvider

Reply:
So ? I change only this one ? All of the file ?

Reply:
:man_shrugging: i'd say at least new code/lines should follow CS standards.

Reply:
It makes sense, done :)